src/test_helpers/tests/gold_stream_evaluator_test.cc

```diff
@@ -0,0 +1,168 @@
+#include "gtest/gtest.h"
+
+#include <memory>
+#include <iostream>
+#include <string>
+#include <exception>
+
+#include "../gold_stream_evaluator.h"
+
+class GoldStreamEvaluatorTest : public ::testing::Test {
+ protected:
+  virtual void SetUp() {
+    gold_iss = std::make_unique<std::istringstream>();
+    actual_iss = std::make_unique<std::istringstream>();
+  }
```

Comment:
Please do not define functions in class declarations. Separate it out and make it much more clear.